go/release-checklist.md

```diff
@@ -66,7 +66,7 @@ Quay release:
 {%- endfor %}
 
 RHCOS packaging for the current RHCOS development release:
- - [ ] Update the spec file
+ - [ ] Update the spec file in [GitLab](https://gitlab.com/redhat/rhel/rpms/{{ gitlab_repo }})
```

Comment:
Hmm, I'd tend to drop `in GitLab` as irrelevant and linkify `spec file` instead.

Reply:
Okay, yeah that sounds good.